sirius_signal_lights_driver/sirius_signal_lights_driver/sent_msgs.py

```diff
@@ -0,0 +1,73 @@
+import rclpy
+from rclpy.node import Node
+from std_msgs.msg import String
+from can_msgs.msg import Frame
+from industrial_msgs.msg import RobotStatus, TriState, RobotMode
+
+
+class SentCanbusMessages(Node):
+    def __init__(self):
+        super().__init__('sent_msgs')
+        self.declare_parameters(
+            namespace='',
+            parameters=[
+                ('device_id', 0),
+                ('command_id', 0),
+                ('green', 0),
+                ('yellow', 0),
+                ('red', 0),
+                ('blue', 0),
+                ('beeper', 0)
+        ])
+        self.my_data = 0
+
+        self.send_topic = self.declare_parameter('send_topic', '/sent_canbus_messages').value
+        self.subscription = self.create_subscription(RobotStatus, '/robot_status', self.get_robot_status, 10)
+        self.subscription
+        self.publisher = self.create_publisher(Frame, self.send_topic, 10)
+        self.my_device_id = self.get_parameter('device_id').value
+        self.my_command_id = self.get_parameter('command_id').value
+        self.my_green = str(self.get_parameter('green').value)
+        self.my_yellow= str(self.get_parameter('yellow').value)
+        self.my_blue = str(self.get_parameter('blue').value)
+        self.my_red = str(self.get_parameter('red').value)
+        self.my_beeper = str(self.get_parameter('beeper').value)
```

Comment:
Why there is `my_` prefix everywhere? 
```suggestion
        self.device_id = self.get_parameter('device_id').value
        self.command_id = self.get_parameter('command_id').value
        self.green = str(self.get_parameter('green').value)
        self.yellow= str(self.get_parameter('yellow').value)
        self.blue = str(self.get_parameter('blue').value)
        self.red = str(self.get_parameter('red').value)
        self.beeper = str(self.get_parameter('beeper').value)
```